app/controllers/assignments_controller.rb

```diff
@@ -6,6 +6,7 @@ class AssignmentsController < ApplicationController
   before_action :set_group
   before_action :check_access, only: %i[edit update destroy reopen]
   after_action :check_reopening_status, only: [:update]
+  after_action :allow_iframe, only: %i[show]
```

Comment:
rename to `allow_iframe_lti` as it is more explicit

Reply:
done